src/main/java/net/snowflake/ingest/utils/Utils.java

```diff
@@ -411,4 +411,22 @@ public static String getFullyQualifiedChannelName(
       String dbName, String schemaName, String tableName, String channelName) {
     return String.format("%s.%s.%s.%s", dbName, schemaName, tableName, channelName);
   }
+
+  /*
+   * Get concat dot path, ignore null or empty string
+   *
+   * @param path the path
+   */
+  public static String concatDotPath(String... path) {
+    StringBuilder sb = new StringBuilder();
+    for (String p : path) {
+      if (!isNullOrEmpty(p)) {
```

Comment:
curious why we want to silently ignore a null or empty path element? Shouldn't we be validating this when we receive the schema from service, and throw if any element is null or empty?

Reply:
Added is null or empty check here.